content/300-guides/100-performance-and-optimization/100-prisma-client-transactions-guide.mdx

```diff
@@ -838,47 +838,65 @@ generator client {
 
 Then you can pass an async function into [$transaction](https://www.prisma.io/docs/guides/performance-and-optimization/prisma-client-transactions-guide/#transaction-api).
 
-The following is an example of purchasing a movie ticket online. This is a good use case for an interactive transactions because you don't want to sell the same ticket twice.
+In the example below, Alice and Bob each have $100 in their account. If you try to send more money than you have, the transfer is rejected.
 
-```ts highlight=6,8,25;normal
-import { Seat } from 'prisma'
-const userEmail = 'alice@prisma.io'
-const movieName = 'HiddenFigures'
-
-// Start a transaction
-const seat: Seat = await prisma.$transaction(async (prisma) => {
-  // Find an available seat
-  const availableSeat = await prisma.seat.findFirst({
-    where: {
-      claimedBy: null,
-      movie: {
-        name: movieName,
+We'd expect Alice to be able to make 1 transfer for $100 and the other transfer would be rejected. This would result in Alice having $0 and Bob having $200.
```

Comment:
```suggestion
The expected outcome would be for Alice to make 1 transfer for $100 and the other transfer would be rejected. This would result in Alice having $0 and Bob having $200.
```